runtime/lua/vim/lsp/codelens.lua

```diff
@@ -199,6 +206,9 @@ end
 
 ---@private
 local function resolve_lenses(lenses, bufnr, client_id, callback)
+  if not api.nvim_buf_is_valid(bufnr) then
+    return
+  end
```

Comment:
I think this still needs to call the callback to ensure the `active_refreshes`  table entry is removed